content/desktop/settings/mac.md

```diff
@@ -89,12 +89,27 @@ Advanced settings are:
 
 - **Disk image location**. Specify the location of the Linux volume where containers and images are stored.
 
-You can also move the disk image to a different location. If you attempt to move a disk image to a location that already has one, you are asked if you want to use the existing image or replace it.
+  You can also move the disk image to a different location. If you attempt to
+  move a disk image to a location that already has one, you are asked if you
+  want to use the existing image or replace it.
 
->**Tip**
+- **Resource Saver**. Enable or disable Resource Saver mode, which significantly
+  reduces CPU and memory utilization on the host by automatically turning off
+  the Linux VM when Docker Desktop is idle (i.e., no containers are running).
+  Exit from Resource Saver mode occurs automatically when containers run.
+
+  You can also configure the Resource Saver timeout which indicates how long
+  should Docker Desktop be idle before Resource Saver mode kicks in. Default is
+  5 minutes.
+
+>**Tips**
 >
-> If you're running multi-container workloads, increase the memory and disk image space allocation to something higher.
-{ .tip } 
+> * Exit from Resource Saver mode occurs automatically when containers run. Exit
```

Comment:
This reads more like a note callout to me...something the user should be aware of, rather than take action with. WDYT?

Reply:
Yes that's fine; my intent was to ensure that it's easily visible to the reader. I'll change it to a note callout.